src/main/java/mate/academy/spring/service/mapper/CinemaHallDtoMapper.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.spring.service.mapper;
+
+import mate.academy.spring.dto.CinemaHallRequestDto;
+import mate.academy.spring.dto.CinemaHallResponseDto;
+import mate.academy.spring.model.CinemaHall;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CinemaHallDtoMapper {
+
+    public CinemaHallResponseDto parse(CinemaHall cinemaHall) {
```

Comment:
remove the redundant empty lines and it would be better to use toDto instead of parse
```suggestion
    public CinemaHallResponseDto parse(CinemaHall cinemaHall) {
```